crates/rattler_repodata_gateway/src/fetch/mod.rs

```diff
@@ -36,6 +48,9 @@ pub enum FetchRepoDataError {
     #[error(transparent)]
     FailedToDownloadRepoData(std::io::Error),
 
+    #[error("repodata not found")]
+    NotFound(#[source] RepoDataNotFoundError),
```

Comment:
If you change `#[source]` to `#[from]` the `From` implementation will be generated for you!

```suggestion
    NotFound(#[from] RepoDataNotFoundError),
```

Reply:
changed